data/test/graves6.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<?teipublisher odd="graves.odd" template="letter.html"?>
+<?teipublisher odd="graves.odd" template="letter.html" media="latex epub print fo"?>
```

Comment:
In the other examples the values are comma-delimited. This is space-delimited. I'm not sure if it makes a difference?